### DISCUSSION TITLE
template tags and filters catch

### DISCUSSION THREAD

**Discussion by Sim4n6** [github/codeql#12312] [Q&A] [answered]
Can we identify the built-in template tags and filters, such as the **safe** use in  `{{ var|safe }}` in Django templates ? [ref](https://docs.djangoproject.com/en/4.1/ref/templates/builtins/#built-in-template-tags-and-filters) ?

**Comment by yoff** [comment, marked as answer]
No, we do not support those templates yet, sorry.

**Reply by Sim4n6** [reply]
Thanks
